pkg/lib.go

```diff
@@ -170,7 +190,23 @@ func GetInventoryReport(cfg *config.Application) (inventory.Report, error) {
 		return inventory.Report{}, fmt.Errorf("failed to get Cluster Server Version: %w", err)
 	}
 
+	var nodes []inventory.Node
+	for _, node := range nodeMap {
+		nodes = append(nodes, node)
+	}
+
 	log.Infof("Got Inventory Report with %d containers running across %d namespaces", len(containers), len(namespaces))
+	if cfg.Metadata {
```

Comment:
what is the difference between this clause and the inventory.Report below it?

Reply:
Nodes are omitted completely if metadata=false, pods & namespaces use the same data structure if metadata is true/false but the contents are reduced by previous calls